lib/transcoder.ex

```diff
@@ -1,5 +1,23 @@
 defmodule Membrane.Transcoder do
-  @moduledoc false
+  @moduledoc """
+  Provides transcoding capabilities for audio and video streams in Membrane.
+
+  The bin takes an incoming stream on its input and converts it into the desired one
+  as specified by the option. Transcoding is applied only if it is neccessary.
+  The following video stream formats are supported:
+  * `Membrane.H264`
+  * `Membrane.H265`
+  * `Membrane.VP8`
+  * `Membrane.RawVideo`
+
+  The following audio stream formats are supported:
+  * `Membrane.AAC`
+  * `Membrane.Opus`
+  * `Membrane.RawAudio`
+  * `Membrane.RemoteStream{content_type: Membrane.Opus}`
+
+  Please note that not all the conversions are possible!
```

Comment:
What conversions are not possible?

Reply:
H264 to AAC :p

Reply:
So I would just write that if the input stream format is audio, the output stream format must be audio as well (this same if the type is video). The way it is written right now suggests that e.g. we might not be able to transcode VP8 to H264

Reply:
Well, perhaps I will just remove that note, the more I think about it the more I am inclining towards conclusion that it is obvious :D